build.rs

```diff
@@ -168,11 +176,14 @@ fn use_static() {
     println!("cargo:rustc-link-search={}", static_ruby_location());
     let static_name = static_ruby_file_name();
     let static_name = Path::new(&static_name).file_stem().unwrap().to_string_lossy();
-    println!("cargo:rustc-link-lib={}", static_name);
+    println!("cargo:rustc-link-lib={}", static_name.trim_left_matches("lib"));
```

Comment:
Not sure if this causes a problem with other platforms, but the `ld` linker expects the arg to `-l` to not be prefixed with `lib`, otherwise: 

```
ld: library not found for -llibruby.2.5.3-static
```

Reply:
The Linux static build test works with this so it should be fine: https://travis-ci.org/danielpclark/rutie/jobs/465793207